sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaSinkMetrics.java

```diff
@@ -71,6 +75,48 @@ public static Histogram createRPCLatencyHistogram(RpcMethod method, String topic
     return new DelegatingHistogram(metricName, buckets, false, true);
   }
 
+  /**
+   * Creates an Gauge metric to record per partition backlog. Metric will have name:
```

Comment:
Nit: `an Gauge` -> `a {@link Gauge}`, ". Metric will have name:" -> " with the name:"

Reply:
Thanks, updated the working for the histogram metric above as well to be consistent.